Add --failOnFailedSeed checkbox to URL list workflows (#1236)
webrecorder/browsertrix: docs/user-guide/workflow-setup.md

??? example "Crawling tags & search queries with URL List crawls"
    This setting can be useful for crawling the content of specific tags or searh queries. Specify the tag or search query URL(s) in the _List of URLs_ field, e.g: `https://example.com/search?q=tag`, and enable _Include Any Linked Page_ to crawl all the content present on that search query page.

> ### Stop Crawl on Failed URL

Reviewer: ```suggestion
### Fail Crawl on Failed URL
```

`URL List`{ .badge-blue }

> When enabled, the crawler will stop crawling if any of the provided URLs are invalid or unsuccessfully crawled. The resulting archived item will have a status of "Partial Complete".

Reviewer: ```suggestion
When enabled, the crawler will fail the entire crawl if any of the provided URLs are invalid or unsuccessfully crawled. The resulting archived item will have a status of "Failed".
```

Reviewer 2: ```suggestion
When enabled, the crawl will fail if any of the provided URLs are invalid or unsuccessfully crawled.
```
I think we can tighten this up further now that the status is actually "failed"!  Should probably dupe to the in-app help text.
